bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCommandWorkItem.java

```diff
@@ -200,13 +200,36 @@ public Collection<WorkItem> prRun(ScratchArea scratchArea) {
         var hostedRepositoryPool = new HostedRepositoryPool(seedPath);
 
         CensusInstance census;
+        var command = nextCommand.get();
+
         try {
             census = CensusInstance.createCensusInstance(hostedRepositoryPool, bot.censusRepo(), bot.censusRef(), scratchArea.getCensus(), pr,
                     bot.confOverrideRepository().orElse(null), bot.confOverrideName(), bot.confOverrideRef());
         } catch (InvalidJCheckConfException | MissingJCheckConfException e) {
-            throw new RuntimeException(e);
+            var writer = new StringWriter();
+            var printer = new PrintWriter(writer);
+
+            printer.println(String.format(COMMAND_REPLY_MARKER, command.id()));
+            printer.print("@");
+            printer.print(command.user().username());
+            printer.print(" ");
+            if (bot.confOverrideRepository().isEmpty()) {
+                var branchNames = pr.repository().branches().stream().map(HostedBranch::name).toList();
+                if (branchNames.contains(pr.targetRef())) {
+                    printer.println("JCheck configuration is missing or invalid in the target branch of this pull request.");
```

Comment:
We do know if it's missing or invalid based on the type of the exception. I think it's worth differentiating the message based on this.